astropy/time/tests/test_basic.py

```diff
@@ -937,11 +941,14 @@ def test_decimalyear():
 
 def test_fits_year0():
     t = Time(1721425.5, format='jd')
-    assert t.fits == '0001-01-01T00:00:00.000'
+    with pytest.warns(ErfaWarning):
```

Comment:
I'd be tempted for these instead to use `scale='tai'` so we do not get warnings as things are well-defined.

Reply:
@mhvk , do you mean just for this test? Or everywhere else that `ErfaWarning` is seen?

Reply:
Ah, nvm, I think you meant this test only, now that I see the other comment below.

Reply:
Done.